### DISCUSSION TITLE
Attribute Validation (`validation`) Function Boolean Return Value Confusing

### DISCUSSION THREAD

**Discussion by bestickley** [tywalch/electrodb#411] [Q&A] [answered]
Does anyone else find the `validation` boolean return signature confusing? See docs [here](https://electrodb.dev/en/modeling/attributes/#attribute-validation). I'd expect returning `true` to be valid and returning `false` to be invalid (like Zod's [refine](https://zod.dev/?id=refine)) but it's the opposite. Am I missing something?

**Comment by tywalch** [comment, marked as answer]
This is likely a good breaking change for [v3](https://github.com/tywalch/electrodb/discussions/424) 👍
